packages/babel-parser/test/fixtures/es2015/arrow-functions/.inner-parens-array-pattern-2/input.js

```diff
@@ -0,0 +1 @@
+([ [(a)] = [] ] = []) => {}
```

Comment:
This test is skipped because we lost the `(a)` error on casting Assignment Expression`[(a)] = []` to Assignment Pattern. The `main` branch does not throw either.

Reply:
Addressed in 00a776c.